.circleci/config.yml

```diff
@@ -89,6 +89,21 @@ jobs:
           working-directory: ./android
           test-command: ./gradlew test
 
+  validate_shell_files:
+    executor:
+      name: android/android-machine
+      tag: "2022.03.1"
```

Comment:
We can run it on a linux machine instead? What do you think? That would be relatively faster.